Source/Fuse.Common/Internal/iOSSystemFont.uno

```diff
@@ -120,21 +184,40 @@ namespace Fuse.Internal
 			var result = new string[descriptors.Length];
 			for (int i = 0; i < descriptors.Length; ++i)
 			{
-				result[i] = GetDescriptorFontName(descriptors[i]);
+				result[i] = GetDescriptorName(descriptors[i]);
 			}
 			return result;
 		}
 
 		[Foreign(Language.ObjC)]
 		static string GetDescriptorFontName(ObjC.Object descriptor)
+		@{
+			return ([UIFont fontWithDescriptor: ((UIFontDescriptor*)descriptor) size: 16]).fontName;
+		@}
+
+		[Foreign(Language.ObjC)]
+		static string GetDescriptorName(ObjC.Object descriptor)
 		@{
 			return ((UIFontDescriptor*)descriptor).fontAttributes[UIFontDescriptorNameAttribute];
 		@}
 
 		[Foreign(Language.ObjC)]
 		static string GetDefaultFontFamily()
 		@{
-			return [UIFont systemFontOfSize: 12].familyName;
+			return [UIFont systemFontOfSize: 16].familyName;
+		@}
+
+		[Foreign(Language.ObjC)]
+		static ObjC.Object GetDefaultFontDescriptor(string weight, bool isItalic)
+		@{
+			NSDictionary* attributes
+				= \@{ UIFontDescriptorFamilyAttribute: [UIFont systemFontOfSize: 12].familyName };
```

Comment:
... but here the font size is 12 again..?

Reply:
I standardised it at 16 (believe me you won't see the default size get bigger cos the defaults always been 16 😄)

Reply:
I see it now. ;)